<html>

<head>
	<title>Research information</title>
	<meta name="description" content="Research interests of Malachi Griffith, BSc, PhD">
	<meta name="keywords" content="Malachi Griffith, Genome Sciences Center, BC Cancer Agency, Contact Details, ALEXA-Seq, ALEXA-WTSS, WTSS, RNA-Seq, Illumina sequencing, Genomics, Bioinformatics, Cancer Research, CV, Resume">
	<link rel="stylesheet" type="text/css" href="MalachiGriffith.css">
</head>

<body>
<!-- Top bar image table -->
<table cellpadding="0" cellspacing="0" width="750" class="Top">
	<tr>
		<td><img src="images/TopRow2.jpg" width="750" height="108" alt="ALEXA logo and images of BC Cancer Agency"></td>
	</tr>
</table>

<!-- Button bar table -->
<table cellpadding="0" cellspacing="0" width="750" class="Middle">
	<tr>
		<td><img src="images/ButtonRowLeftSpacer2.jpg" width="324" height="27" alt=""></td>
		<td><a href="index.html"><img src="images/Top_Home_black.jpg" border="0" width="56" height="27" alt="Button link to homepage of the ALEXA-Seq website"></a></td>
		<td><a href="research.htm"><img src="images/Top_Research_white.jpg" border="0" width="79" height="27" alt="Button link to an introduction to the ALEXA-Seq website"></a></td>
		<td><a href="funding.htm"><img src="images/Top_Funding_black.jpg" border="0" width="70" height="27" alt="Button link to detailed methods for how ALEXA-Seq is used to study alternative transcript expression"></a></td>
		<td><a href="publications.htm"><img src="images/Top_Publications_black.jpg" border="0" width="102" height="27" alt="Button link to results for a validation experiment using the ALEXA-Seq approach"></a></td>
		<td><a href="cv.htm"><img src="images/Top_CV_black.jpg" border="0" width="32" height="27" alt="Button link to the download area for supplementary data and source code"></a></td>
		<td><a href="contact.htm"><img src="images/Top_Contact_black.jpg" border="0" width="71" height="27" alt="Button link to the download area for supplementary data and source code"></a></td>
		<td><img src="images/ButtonRowSpacerRight.jpg" width="16" height="27" alt=""></td>
	</tr>
</table>

<!-- Bar under buttons table -->
<table cellpadding="0" cellspacing="0" width="750" class="Middle">
	<tr>
		<td><img src="images/BarUnderButtons.jpg" width="750" height="32" alt=""></td>
	</tr>
</table>

<!--- expandable content section -->
<table cellpadding="0" cellspacing="0" width="752" class="Content">
	<tr>
		<td>
		<br>
		<p class="Indented12LR_s16_Bold">Research Interests</p>
		<br>
		<p class="Indented12LR_bold">Genomics and Cancer Medicine</p>
		<p class="Indented12LR">My PhD research focused on genome-wide analysis of alternative mRNA isoforms (arising by alternative splicing, initiation and poly-adenylation) 
		[<a href="files/GGG_Chapter12_GriffithAndMarra_2007.pdf">ref</a>].  
		The principle goal of this work was to develop computational and genomic methods to identify those mRNA isoforms that represent potential therapeutic, diagnostic, 
		and prognostic biomarkers in cancer. This goal was initially pursued by creating a computational platform for the design and analysis of splicing sensitive microarrays 
		for alternative expression analysis ('ALEXA-arrays') [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18235430">ref</a>].  
		Subsequently, the advent of massively parallel RNA sequencing technology allowed me to develop 'ALEXA-Seq' [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20835245">ref</a>] 
		and apply this approach to almost <a href="http://www.alexaplatform.org/alexa_seq/results.htm">20 models</a> of cancer progression and tissue development. 
		Both the splicing microarray and RNA sequencing approaches were published and associated software and databases were made available at our website: 
		<a href="http://www.AlexaPlatform.org">www.AlexaPlatform.org</a>.  
		The utility of these approaches was demonstrated by discovery and validation of a novel isoform associated with fluorouracil resistance in a panel of pre- and post-treatment 
		colorectal cancer tumors.  Although my PhD thesis work focused on development and application of bioinformatic methods for alternative expression analysis, I am interested 
		in many genomic and informatics approaches to studying cancer biology and medicine.  I have participated in analysis and identification of: 
		alternative mRNA isoforms [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20835245">ref</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/18235430">ref2</a>],
		miRNAs [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18285502">ref</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/18849523">ref</a>],
		copy number variations [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18559357">ref</a>], 
		point mutations [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20696054">ref</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/17010196">ref</a>], 
		insertions/deletions, 
		structural rearrangements [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20935650">ref</a>], 
		and fusion genes [<a href="http://compbio.bccrc.ca/?page_id=275">ref</a>] associated with cancer.  During the course of these collaborations I gained experience analyzing 
		data from: Sanger sequencing of full-length cDNA clones, Sanger sequencing of genomic PCR amplicons, Affymetrix U133 gene expression arrays, Affymetrix exon expression arrays, custom NimbleGen 
		splicing microarrays (designed by me), Affymetrix SNP 6.0 arrays (for copy number assessment), Roche 454 RNA sequencing, Illumina GAII RNA sequencing, 
		Illumina GAII exon capture sequencing, and Illumina HiSeq RNA sequencing.
		</p><br>
		
		<p class="Indented12LR_bold">Bioinformatics</p>
		<p class="Indented12LR">Many genomics and informatics tasks in cancer research involve using and adapting a combination of existing algorithms, tools, databases, and
		parallel computing.  I have also participated in the development and application of novel bioinformatic methods including creating software and databases where 
		existing approaches have fallen short. I have experience programming in Unix, Perl, SQL, Bash, BASIC, C, C++, HTML, CFML, and the statistical programming language R.  
		I have recent experience with: ALEXA-seq, Cufflinks, Scripture, Tophat, Bowtie, Picard, HMMSplicer, Maq, BWA, NovoAlign, BEDtools, Velvet, Galaxy, the EnsEMBL Perl API, 
		Xapian/Omega, Vector NTI, Consed, PhredPhrap, BLAST, BLAT, clustalW, Bioconductor, Affymetrix’s Expression Console and ExACT, Agilent’s eArray, mdust, RNAfold, 
		pairFold, mySQL, custom UCSC tracks, VMware virtual machines, and countless other commonly used bioinformatics tools.
		</p><br>
		
		<p class="Indented12LR_bold">Alternative mRNA expression</p>
		<p class="Indented12LR">Initial studies following from the Human Genome Project have revealed that the apparent number of genes present in a human is much less 
		than expected for such a complex organism. Recent studies suggest that in fact it is not just the number of genes that gives rise to our complexity but 
		rather the number of functionally distinct versions of a gene that can be encoded from a single gene region. Human genes are comprised of DNA sequences called exons 
		separated by long stretches of DNA sequence called introns which must be removed so that the exons can be assembled into a working copy of the gene. My PhD research 
		focused on the phenomenon of alternative splicing in which one gene is assembled from its component pieces in many different ways to produce a multitude of different 
		functional products. In particular, changes in the forms of certain genes may be important in the progression of cancer and account for the differences in the severity 
		of cancers and response to treatment observed among individuals. By studying the alternative splicing of genes in models of cancer the hope is to 
		identify promising candidates for vaccine and drug development. 
		</p><br>
		
		<p class="Indented12LR_bold">Custom splicing microarrays for alternative expression analysis</p>
		<p class="Indented12LR">The use of microarray technology to profile mRNA transcripts generated by alternative splicing is an area of rapid development. To facilitate 
		the use of microarrays to study alternative splicing I created an open source array design platform for alternative expression analysis called ‘ALEXA' 
		(<a href="http://www.AlexaPlatform.org">www.AlexaPlatform.org</a>). 
		This platform allows the design and analysis of microarrays of arbitrary density and complexity for alternative expression 
		analysis of most EnsEMBL annotated species.  Creation of ALEXA arrays involves the extraction, scoring, filtering and annotation of oligonucleotide probes 
		corresponding to exons, introns, exon boundaries and exon-exon junctions. I used this platform to pre-compute designs for ten EnsEMBL annotated species. 
		To evaluate the ALEXA-array approach I generated a design for the human genome and measured differential expression of alternate isoforms between 5-fluorouracil 
		sensitive and resistant colorectal cancer cell lines. Results generated from ALEXA arrays were compared to those from Affymetrix exon arrays.  ALEXA array data 
		was comparable or superior to Affymetrix exon arrays in terms of reproducibility, sensitivity and specificity and provided additional information on the connectivity 
		and boundaries of exons.  Analysis results, microarray designs and source code for ALEXA-arrays are available at 
		<a href="http://www.alexaplatform.org/alexa_arrays/">http://www.alexaplatform.org/alexa_arrays/</a>.
		</p><br>
		
		<p class="Indented12LR_bold">Alternative expression analysis by massively parallel RNA sequencing</p>
		<p class="Indented12LR">In alternative expression analysis by sequencing (ALEXA-seq), I developed a method to analyze massively parallel RNA sequence data to 
		catalog transcripts and assess differential and alternative expression of known and predicted mRNA isoforms in cells and tissues. As proof of principle, I used 
		the approach to compare fluorouracil-resistant and -nonresistant human colorectal cancer cell lines. I assessed the sensitivity and specificity of the approach 
		by comparison to exon tiling and splicing microarrays and validated the results with reverse transcription-PCR, quantitative PCR and Sanger sequencing. I observed 
		global disruption of splicing in fluorouracil-resistant cells characterized by expression of new mRNA isoforms resulting from exon skipping, alternative splice site 
		usage and intron retention. Alternative expression annotation databases, source code, a data viewer and other resources to facilitate analysis are available at 
		<a href="http://www.alexaplatform.org/alexa_seq/">http://www.alexaplatform.org/alexa_seq/</a>.
		</p><br>
		
		<p class="Indented12LR_bold">Mechanisms of chemotherapy resistance in colorectal cancer</p>
		<p class="Indented12LR">Colorectal cancer (CRC) is the 4th most commonly diagnosed type of cancer and has the 2nd highest number of fatalities in Canada 
		(2007 Canadian Cancer Statistics). Treatment commonly involves surgery and adjuvant or neo-adjuvant systemic chemotherapy.  One of the most commonly used 
		chemotherapy drugs in the treatment of CRC is the uracil analog, 5-fluorouracil (5-FU).  The high mortality rate associated with CRC is due in part to resistance 
		to 5-FU treatment.  The factors that determine 5-FU efficacy are complex and it is likely that any clinical test of 5-FU response that is based on the characterization 
		of only one or two of the many genes involved will fall short of being applicable to the general population.  Despite its common usage, the response rate to 5-FU is 
		low (as low as 20% when used as a single agent).  While for some patients these drugs are highly effective, for others they either have no favourable response or 
		an adverse reaction.  Developing an accurate means to identify responders and determine appropriate dosage is critical to improving this situation. If we can understand 
		what causes resistance to 5-FU, we can use this information to avoid giving the drug to patients that are unlikely to benefit from it.  Furthermore, if we can understand 
		what causes resistance to drugs, we can use this information to design new drugs to combat or avoid this resistance.  
		To pursue this goal we developed and applied genomic methods to profile the expression, RNA processing and sequence polymorphisms of human transcripts and to identify 
		those events associated with resistance to 5-FU in colorectal cancer. We selected a series of colon cancer cell lines representing a cancer that is 
		sensitive to chemotherapy or resistant to one of four commonly used chemotherapy drugs. By studying the differences in gene structure correlated with this change in 
		drug response we gained insight into why chemotherapy initially seems to work well in some patients but becomes less effective over time.  This study represents one 
		contribution to the ultimate goal of one day tailoring cancer treatments to the needs of individual patients and lessening the incidence, morbidity and mortality 
		from cancer. 
		</p><br>

		<p class="Indented12LR_bold">Identifying, characterizing, and processing tumors for clinical validation studies</p>
		<p class="Indented12LR">During the course of participating in genomic and bioinformatic analysis of cancer, I became interested (out of necessity) in many of the
		technical and logistical challenges faced in attempting clinical validation of our predictions.  This has lead me to a greater appreciation of the complexity and importance 
		of integrated teams in cancer genomics research.  My own experience has involved working with students, scientists, surgeons, oncologists, pathologists and hospital and 
		biobank administrators.  I have gained experience in the principles and writing of ethics applications.  I have found that exposure to patient records, such as oncology, 
		surgical, and pathology reports is invaluable in gaining perspective on the treatment and outcomes of a particular cancer.  I also gained an appreciation for the art of 
		isolating nucleic acid material from patient samples and how this contrasts with isolation from tissue culture sources.  This appreciation was gained by extensive 
		experimentation with RNA and DNA isolation from cultured cells, fresh frozen archival samples from tumour banks, and formalin fixed paraffin embedded (FFPE) tumor blocks 
		from several anatomical pathology departments across the province.  Ultimately I used my experience to train and supervise students to assist in these activities, 
		without whom obtaining the hundreds of samples needed for our studies would not have been possible.
		</p><br>
		
		<br>
		</td>
	</tr>
</table>

<!-- Bottom button row -->
<table cellpadding="0" cellspacing="0" width="750" class="Bottom">
	<tr>
		<td><img src="images/BottomRowSpacerLeft.jpg" width="20" height="25" alt=""></td>
		<td><a href="index.html"><img src="images/Bottom_Home_black.jpg" width="60" height="25" border="0" alt="Button link to main ALEXA home page"></a></td>
		<td><a href="research.htm"><img src="images/Bottom_Research_white.jpg" width="87" height="25" border="0" alt="Button link to ALEXA-Arrays home page"></a></td>
		<td><a href="funding.htm"><img src="images/Bottom_Funding_black.jpg" width="81" height="25" border="0" alt="Button link to ALEXA-Seq home page"></a></td>				
		<td><a href="publications.htm"><img src="images/Bottom_Publications_black.jpg" width="103" height="25" border="0" alt="Button link to acknowledgements of funding and other support for Malachi Griffith and Marco Marra"></a></td>
		<td><a href="cv.htm"><img src="images/Bottom_CV_black.jpg" width="33" height="25" border="0" alt="Button link to contact information for Malachi Griffith and Marco Marra"></a></td>
		<td><a href="contact.htm"><img src="images/Bottom_Contact_black.jpg" width="105" height="25" border="0" alt="Button link to contact information for Malachi Griffith and Marco Marra"></a></td>
		<td><img src="images/BottomRowSpacerRight.jpg" width="261" height="25" alt=""></td>
	</tr>
</table>

</body>

</html>
