<html>

<head>
	<title>Research information</title>
	<meta name="description" content="Research interests of Malachi Griffith, BSc, PhD">
	<meta name="keywords" content="Malachi Griffith, Contact Details, RNA-Seq, Genomics, Bioinformatics, Cancer Research, CV, Resume">
	<link rel="stylesheet" type="text/css" href="MalachiGriffith.css">
</head>

<body>
<!-- Top bar image table -->
<table cellpadding="0" cellspacing="0" width="750" class="Top">
	<tr>
		<td><img src="images/TopRow2.jpg" width="750" height="108" alt="logo and images of BC Cancer Agency"></td>
	</tr>
</table>

<!-- Button bar table -->
<table cellpadding="0" cellspacing="0" width="750" class="Middle">
	<tr>
		<td><img src="images/ButtonRowLeftSpacer2.jpg" width="324" height="27" alt=""></td>
		<td><a href="index.html"><img src="images/Top_Home_black.jpg" border="0" width="56" height="27" alt="Button link to homepage"></a></td>
		<td><a href="research.htm"><img src="images/Top_Research_white.jpg" border="0" width="79" height="27" alt="Button link to research"></a></td>
		<td><a href="funding.htm"><img src="images/Top_Funding_black.jpg" border="0" width="70" height="27" alt="Button link to funding"></a></td>
		<td><a href="publications.htm"><img src="images/Top_Publications_black.jpg" border="0" width="102" height="27" alt="Button link to publications"></a></td>
		<td><a href="cv.htm"><img src="images/Top_CV_black.jpg" border="0" width="32" height="27" alt="Button link to cv"></a></td>
		<td><a href="contact.htm"><img src="images/Top_Contact_black.jpg" border="0" width="71" height="27" alt="Button link to contact"></a></td>
		<td><img src="images/ButtonRowSpacerRight.jpg" width="16" height="27" alt=""></td>
	</tr>
</table>

<!-- Bar under buttons table -->
<table cellpadding="0" cellspacing="0" width="750" class="Middle">
	<tr>
		<td><img src="images/BarUnderButtons.jpg" width="750" height="32" alt=""></td>
	</tr>
</table>

<!--- expandable content section -->
<table cellpadding="0" cellspacing="0" width="752" class="Content">
	<tr>
		<td>
		<br>
		<p class="Indented12LR_s16_Bold">Research interests</p>
		<br>
		<p class="Indented12LR_bold">Personal statement</p>
    <p class="Indented12LR">
    I am an Assistant Professor of Medicine and Assistant Director of the McDonnell Genome Institute at Washington University School of Medicine. I have more than 13 years 
    of experience in the fields of genomics, bioinformatics, data mining, and cancer research. My research is focused on improving our understanding of cancer biology and 
    the development of personalized medicine strategies for cancer using genomics and informatics technologies. I develop bioinformatics and statistical methods for the analysis 
    of high throughput sequence data and identification of biomarkers for diagnostic, prognostic and drug response prediction. In collaboration with our Genomics Tumor Board, 
    I am currently leading analyses of sequence data from the genomes and transcriptomes of living cancer patients that have exhausted standard-of-care treatment options. 
    I am also leading the analysis of several large-scale genomics projects to help discover genomic signatures relevant to cancer initiation, progression and treatment response. 
    I am an instructor for the Canadian Bioinformatics Workshops and Cold Spring Harbor Laboratory high-throughput sequencing workshop series. I have received numerous competitive 
    research awards from the Natural Sciences and Engineering Research Council, University of Winnipeg, University of British Columbia, National Cancer Institute of Canada and 
    British Columbia Cancer Agency. I was also a Junior and Senior Trainee of the Michael Smith Foundation for Health Research. I was profiled as an "up and coming" investigator in 
    Genome Technology Magazine and listed in Wired Magazine second annual smart list as an emerging talent. Most recently, I was awarded an NHGRI K99 Pathway to Independence 
    Award to developed integrated analysis and interpretation methods for whole genome, exome and transcriptome sequence data in cancer. In particular, I am focused on applying these 
    methods to better understand the regulatory consequence of somatic mutations that are otherwise of unknown functional significance. I have also made substantial contributions 
    to open source and open access resources for the scientific community including creation of platforms for alternative expression analysis 
    (<a href="http://www.alexaplatform.org/">www.alexaplatform.org</a>), regulatory region annotation (<a href="http://www.oreganno.org">www.oreganno.org</a>), mining drug-gene 
    interactions (<a href="http://www.dgidb.org">www.dgidb.org</a>), genome analysis development (<a href="https://github.com/genome/gms">https://github.com/genome/gms</a>), 
    curation of functional cancer mutations (<a href="http://www.docm.info">www.docm.info</a>), the clinical interpretation of variants in cancer 
    (<a href="http://www.civicdb.org">www.civicdb.org</a>). I have published 39 papers with major areas of focus in precision medicine, AML, colon cancer, lung cancer, cancer genomics, 
    cancer biomarkers, sequence analysis, and alternative splicing. The following paragraphs describe some general contributions to science and provide an overview of my current 
    research interests. For more specific details refer to the <a href="publications.htm">publications</a> section.
		</p><br>
		
		<p class="Indented12LR_bold">Genomics technology</p>
    <p class="Indented12LR">
    For the past 12 years I have worked continuously at two large-scale genome sequencing and analysis centers.  Early access to new technologies has allowed me to develop cutting edge 
    approaches for genome, transcriptome, and epigenome analysis.  For example, I helped to develop some of the first massively parallel sequencing assays for microRNAs, alternative 
    expression analysis of RNA-seq data, copy number variant analysis by tumor whole genome sequencing, RNA-seq and cDNA capture of challenging archival (FFPE) clinical samples, and 
    epigenetic profiling by ChIP-seq.
		</p><br>
		
		<p class="Indented12LR_bold">Alternative splicing and gene regulation</p>
    <p class="Indented12LR">
    An area of particular focus in my research has been the development of new massively parallel assays to study alternative expression (alternative transcript initiation, splicing, 
    and polyadenylation). Following on from early work in the Mammalian Gene Collection project to produce a catalogue of all human transcripts, I developed a custom microarray design 
    platform for alternative expression analysis (ALEXA, <a href="http://www.alexaplatform.org/alexa_arrays/">www.alexaplatform.org/alexa_arrays/</a>). With the advent of massively 
    parallel sequencing, I adapted the principles and analysis methods learned from my work with splicing microarrays to create an alternative splicing analysis platform ALEXA-seq to 
    mine RNA-seq data (<a href="http://www.alexaplatform.org/alexa_seq/">www.alexaplatform.org/alexa_seq/</a>). I have since applied these approaches to address several specific 
    biological questions relating to the mechanisms of RNA transcript processing and aberrant RNA splicing observed in tumors, particularly blood cancers where mutations in certain 
    splicing factors such as U2AF1 are common.
		</p><br>
    
    <p class="Indented12LR_bold">Bioinformatics</p>
    <p class="Indented12LR">
    In addition to tools and algorithms developed for analysis of RNA-seq data in the context of alternative splicing I have helped to release many other bioinformatic tools, databases, 
    and web resources including tools with very focused applications such as <a href="https://github.com/genome/sciclone">SciClone</a>, <a href="http://www.alexaplatform.org/alexa_seq/">ALEXA-Seq</a>
    and <a href="http://www.dgidb.org">DGIdb</a> but also resources with much broader application including the <a href="https://github.com/genome/gms">Genome Modeling System</a> and 
    an RNA-seq analysis resource for cloud computing (<a href="http://www.rnaseq.wiki">www.rnaseq.wiki</a>).  I am currently co-leading the development of the first completely open-source 
    and open-access resource to define a community consensus on the clinical interpretation of variants in cancer (<a href="http://www.civicdb.org">www.civicdb.org</a>).  A theme of all 
    of these efforts is transparent development including version tracked source code and unrestrictive licensing of both the code and the content produced so that the community can benefit 
    most efficiently from this work.
		</p><br>
    
    <p class="Indented12LR_bold">Cancer biology</p>
    <p class="Indented12LR">
    Developing new methods, assays, algorithms and bioinformatics tools is a means to an end in my research program.  My goal is to apply these to improve our understanding of cancer 
    biology and to translate discoveries into improved outcomes for patients suffering from cancer.  I have applied these new methods to address specific biological questions in 
    several cancer types and treatment scenarios. These studies range from creating the first genomic landscapes of some cancer types such as lung cancer, to pursuing specific candidate 
    biomarkers of therapeutic resistance in colon cancer, to defining the clonal architecture of AML.
		</p><br>

    <p class="Indented12LR_bold">Precision medicine</p>
    <p class="Indented12LR">
    My greatest research interest is personalized or precision medicine. I am broadly interested in the translation of genomics data from whole genome, exome and transcriptome sequencing 
    into clinically actionable observations and personalized cancer therapies. In 2010, I was involved in one of the first attempts to sequence the genome of a patient in real time and 
    personalize therapy based on the analysis of this data. In 2011, I helped to lead the analysis of a second case that lead to a widely reported discovery of aberrant FLT3 over-expression 
    and unique therapeutic intervention in a case of adult ALL. Since those two cases I have continued to focus on the work needed to make clinical genome sequencing of individual patients 
    a reality. To date, I have contributed to the analysis of hundreds of tumors representing more than ten tumor types in collaboration with over a dozen clinicians.
		</p><br>
		
		<br>
		</td>
	</tr>
</table>

<!-- Bottom button row -->
<table cellpadding="0" cellspacing="0" width="750" class="Bottom">
	<tr>
		<td><img src="images/BottomRowSpacerLeft.jpg" width="20" height="25" alt=""></td>
		<td><a href="index.html"><img src="images/Bottom_Home_black.jpg" width="60" height="25" border="0" alt="Button link to home page"></a></td>
		<td><a href="research.htm"><img src="images/Bottom_Research_white.jpg" width="87" height="25" border="0" alt="Button link to research"></a></td>
		<td><a href="funding.htm"><img src="images/Bottom_Funding_black.jpg" width="81" height="25" border="0" alt="Button link to funding"></a></td>				
		<td><a href="publications.htm"><img src="images/Bottom_Publications_black.jpg" width="103" height="25" border="0" alt="Button link to publications for Malachi Griffith and Marco Marra"></a></td>
		<td><a href="cv.htm"><img src="images/Bottom_CV_black.jpg" width="33" height="25" border="0" alt="Button link to CV"></a></td>
		<td><a href="contact.htm"><img src="images/Bottom_Contact_black.jpg" width="105" height="25" border="0" alt="Button link to contact for Malachi Griffith"></a></td>
		<td><img src="images/BottomRowSpacerRight.jpg" width="261" height="25" alt=""></td>
	</tr>
</table>

</body>

</html>
